Dockerfile

```diff
@@ -7,14 +7,11 @@ FROM golang:1.12.7-alpine3.10 AS builder
 
```

Comment:
You should really move from golang 1.12 to the latest version of golang. 

1.14 was released on February 25, 2020. Since golang versions are supported for two major releases (https://endoflife.date/go), this means that Golang 1.12 is not supported anymore - it's passed its end-of-life.